LeaderboardBackend.Test/Features/Users/LoginTests.cs

```diff
@@ -0,0 +1,126 @@
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Models.Requests;
+using LeaderboardBackend.Models.Validation;
+using LeaderboardBackend.Test.Fixtures;
+using LeaderboardBackend.Test.Lib;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace LeaderboardBackend.Test.Features.Users;
+
+public class LoginTests : IntegrationTestsBase
+{
+    private const string LOGIN_URI = "/login";
```

Comment:
Use the constants you defined in consts.cs.

Reply:
What's the point of doing `private const string LOGIN_URI = Routes.LOGIN;`? Just use Routes.LOGIN directly.

Reply:
Why? Because I'm a genius.

(Changing)